streams/src/test/java/org/apache/kafka/streams/integration/ResetIntegrationTest.java

```diff
@@ -187,7 +187,8 @@ public void testReprocessingFromScratchAfterResetWithIntermediateUserTopic() thr
         streams.close();
 
         assertThat(resultRerun, equalTo(result));
-        assertThat(resultRerun2, equalTo(result2));
+        final int maxToCompare = Math.max(result2.size(), resultRerun2.size());
```

Comment:
Should be min -- good catch! (problem: the test passed for me locally all the time, so this "fix" does not change local runs)